src/pages/gateway/create-mesh.md

```diff
@@ -64,27 +51,108 @@ description: Create a configuration file for your mesh, access the gateway, and
+            "name": "Commerce",
+            "handler": {
+              "graphql": {
+                "endpoint": "https://<your_commerce_site>/graphql/"
+              }
+            }
+          },
+          {
+            "name": "AEM",
+            "handler": {
+              "graphql": {
+                "endpoint": "https://<your_AEM_site>/endpoint.json"
+              }
+            }
+          },
+          {
+            "name": "LiveSearch",
+              "handler": {
+                "graphql": {
+                  "endpoint": "https://<your_commerce_site>/search/graphql",
+                  "operationHeaders": {
+                    "x-api-key": "search_gql",
+                    "Magento-Environment-Id": "<your_environment_id>",
+                    "Magento-Website-Code": "base",
+                    "Content-Type": "application/json",
+                    "Magento-Store-Code": "main_website_store",
+                    "Magento-Store-View-Code": "default"
+                  },
+                  "schemaHeaders": {
+                    "x-api-key": "search_gql",
+                    "Magento-Environment-Id": "<your_environment_id>",
+                    "Magento-Website-Code": "base",
+                    "Content-Type": "application/json",
+                    "Magento-Store-Code": "main_website_store",
+                    "Magento-Store-View-Code": "default"
+                  }
+                }
+              }
+            }
+          ]
+        }
+    }
+```
 
-1. Select the project and workspace that you want to create the mesh in. You will be assigned a `meshId`, which is the case-sensitive, readable name you will use to refer to your mesh in the future. Your assigned `meshId` will look something like this: `12a3b4c5-6d78-4012-3456-7e890fa1bcde`.
+## Create an API Key
 
 <InlineAlert variant="info" slots="text"/>
 
-Each workspace within a project can only have one mesh associated with it at a time.
+Only mesh owners can create API Keys. If you do not have access to [Adobe Developer Console], contact your mesh owner.
+
+To access the gateway and perform GraphQL queries, you need to provide an API Key to authorize access to your mesh. To create your API Key:
+
+1. In [Adobe Developer Console], select the desired organization from the dropdown in the top-right corner.
+
+    ![create a project](../_images/create-project.png)
+
+1. Select an existing project or [create a new one](#create-a-project).
+
+1. Inside the project, click **Add API**.
+
+    ![add an api](../_images/add-api.png)
+
+1. Select **API Mesh for Adobe Developer App Builder** and click **Next**.
+
+    ![add an api mesh](../_images/add-api-mesh.png)
+
+1. The **Allowed Domain** field is not currently enforced. Enter any valid test domain to proceed.
+
+    ![add an allowed domain](../_images/allowed-domain.png)
+
+1. Click **Save configured API**. Copy your **API Key** from the Project Overview page.
+
+    ![api key](../_images/api-key.png)
```

Comment:
I can't comment on the png file directly... Should the API key be blurred?

Reply:
Maybe? I have already deleted/disabled that key, but it could be a legitimate key in the future. @revanth0212 what are your thoughts?

Reply:
Good point guys. Let's blur it for safety.

Reply:
Blurred in latest commit